script/hassfest/config_flow.py

```diff
@@ -46,6 +56,7 @@ def validate_integration(config: Config, integration: Integration):
 
     has_unique_id = (
         "self.async_set_unique_id" in config_flow
+        or "self.async_handle_discovery_without_unique_id" in config_flow
```

Comment:
We're doing string checks here, I don't think that any implementation has this because it is all inherited from `ConfigFlow` ?

Reply:
And if it is overridden to add more logic to it?

Either `async_set_unique_id` or the `async_handle_discovery_without_unique_id` handle it correctly, another other method won't.

I agree the use case if more limited.